doc/sphinx/language/core/conversion.rst

```diff
@@ -35,35 +35,34 @@ We say that :math:`\subst{t}{x}{u}` is the *β-contraction* of
 :math:`((λx:T.~t)~u)` and, conversely, that :math:`((λ x:T.~t)~u)` is the
 *β-expansion* of :math:`\subst{t}{x}{u}`.
 
-According to β-reduction, terms of the *Calculus of Inductive
-Constructions* enjoy some fundamental properties such as confluence,
+According to :gdef:`β-reduction <beta-reduction>`, terms of the *Calculus of
```

Comment:
The actual definition of the term is the sentence above "We define for this a reduction rule we call beta".
The current paragraph doesn't define anything and I'm thinking "According" is not even the appropriate term to use here (probably "Thanks to" would be more appropriate). Anyway, as you said in  #13811, this section is very poorly written and would deserve a big update.

Reply:
I removed a few words, a band-aid for now.